dist/images/kubectl-ko

```diff
@@ -460,6 +468,141 @@ dbtool(){
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            kubectl cp /tmp/kube_ovn_fastpath.ko kube-system/"$i":/tmp/
+          done
+          ;;
+        *)
+          echo "unknown system $sys"
+      esac
+      ;;
+    remove-fastpath)
+      case $sys in
+        centos)
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            kubectl -n kube-system exec "$i" -- rm -f /tmp/kube_ovn_fastpath.ko
+          done
+          ;;
+        *)
+          echo "unknown system $sys"
+      esac
+      ;;
+    install-stt)
+      case $sys in
+        centos7)
+          # shellcheck disable=SC2145
+          docker run -it --privileged -v /lib/modules:/lib/modules -v /usr/src:/usr/src -v /tmp:/tmp kubeovn/centos7-compile:"$OVN_VERSION" bash -c "./module.sh stt install"
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            for k in /tmp/*.rpm; do
+              kubectl cp "$k" kube-system/"$i":/tmp/
+            done
+          done
+          ;;
+        centos8)
+          # shellcheck disable=SC2145
+          docker run -it --privileged -v /lib/modules:/lib/modules -v /usr/src:/usr/src -v /tmp:/tmp kubeovn/centos8-compile:"$OVN_VERSION" bash -c "./module.sh stt install"
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            for k in /tmp/*.rpm; do
+              kubectl cp "$k" kube-system/"$i":/tmp/
+            done
+          done
+          ;;
+        *)
+          echo "unknown system $sys"
+      esac
+      ;;
+    local-install-stt)
+      case $sys in
+        centos7)
+          # shellcheck disable=SC2145
+          docker run -it --privileged -v /lib/modules:/lib/modules -v /usr/src:/usr/src -v /tmp:/tmp kubeovn/centos7-compile:"$OVN_VERSION" bash -c "./module.sh stt local-install $@"
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            for k in /tmp/*.rpm; do
+              kubectl cp "$k" kube-system/"$i":/tmp/
+            done
+          done
+          ;;
+        centos8)
+          # shellcheck disable=SC2145
+          docker run -it --privileged -v /lib/modules:/lib/modules -v /usr/src:/usr/src -v /tmp:/tmp kubeovn/centos8-compile:"$OVN_VERSION" bash -c "./module.sh stt local-install $@"
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            for k in /tmp/*.rpm; do
+              kubectl cp "$k" kube-system/"$i":/tmp/
+            done
+          done
+          ;;
+        *)
+          echo "unknown system $sys"
+      esac
+      ;;
+    remove-stt)
+      case $sys in
+        centos)
+          for i in $(kubectl -n kube-system get pods | grep ovn-cni | awk '{print $1}');
+          do
+            kubectl -n kube-system exec "$i" -- rm -f /tmp/*.rpm
```

Comment:
we should delete file carefully, use might also store other rpm in `/tmp`

Reply:
Now we delete files like "openvswitch-kmod*.rpm"